form-editor-studio-lib/src/main/java/com/tallence/formeditor/studio/validator/field/AbstractFormValidator.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2018 Tallence AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.tallence.formeditor.studio.validator.field;
+
+import com.coremedia.rest.validation.Issues;
+import com.coremedia.rest.validation.Severity;
+import com.tallence.formeditor.contentbeans.FormEditor;
+
+abstract class AbstractFormValidator {
+
+  void addErrorIssue(Issues issues, String formElementId, String propertyName, String errorCode, Object... objects) {
+    String property = "localSettings." + FormEditor.FORM_ELEMENTS + "." + formElementId + "." + propertyName;
```

Comment:
Hier ist in Zeile 93 "formData" eingetragen:
https://github.com/tallence/core-forms/blob/master/form-editor-studio-plugin/src/main/joo/com/tallence/formeditor/studio/studioform/FormEditorForm.mxml
Das sollte also übereinstimmen. Oder alternativ zentral konfiguriert werden können. Da der StructPropertyName aber hier auch extra hinzugefügt wird https://github.com/tallence/core-forms/blob/master/form-editor-server/src/main/resources/framework/doctypes/form-editor-doctypes.xml est das m.E. erstmal nicht notwendig

Reply:
Ist korrigiert, per String Konstante